internal/resources/resource_dhcp_server.go

```diff
@@ -0,0 +1,119 @@
+// (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+
+package resources
+
+import (
+	"context"
+
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/internal/utils"
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/pkg/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DhcpServer() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Provide the DHCP server name",
+			},
+			"network_server_id": {
+				Type:        schema.TypeInt,
+				Description: "NSX-T Integration ID",
+				Computed:    true,
+			},
+			"lease_time": {
+				Type:        schema.TypeInt,
```

Comment:
what is the max value of lease time ? any default value ? i hope it fits in Int

Reply:
yea default value added 86400